apps/bitwarden-registration.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# T&M Hansson IT AB © - 2020, https://www.hanssonit.se/
+
+# shellcheck disable=2034,2059
+true
+# shellcheck source=lib.sh
+. <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh)
+
+# Check for errors + debug code and abort if something isn't right
+# 1 = ON
+# 0 = OFF
+DEBUG=0
+debug_mode
+
+# Must be root
+root_check
+
+# Check if Bitwarden is already installed
+print_text_in_color "$ICyan" "Checking if Bitwarden is already installed..."
+if is_docker_running
+then
+    if docker ps -a --format '{{.Names}}' | grep -Eq "bitwarden";
+    then
+        if [ -d /root/bwdata ]
+        then
+            sleep 0.1
+         else
```

Comment:
instead of sleep 0.1 you could simply use 
```
if [ ! -d /root/bwdata ]
then 
    msg_box "It seems like 'Bitwarden' isn't installed.\n\nYou cannot run this script."
    exit 1
fi
```

Reply:
Duhh, was tired yesterday. Thanks!